src/lualdap.c

```diff
@@ -29,6 +29,9 @@
 #  define luaL_newlib(L,l) (lua_newtable(L), luaL_register(L,NULL,l))
 #  define luaL_setfuncs(L,l,n) (assert(n==0), luaL_register(L,NULL,l))
 #endif
+#if LUA_VERSION_NUM < 503
+#  define lua_getfield(L, i, k) (lua_getfield((L), (i), (k)), lua_type((L), -1))
```

Comment:
Since b592959c432f31ba3151f2c806e4feee6d69baad this function is available in lualdap, so this definition should now be unnecessary.